.gitmodules

```diff
@@ -19,3 +19,9 @@
 	path = lib/libcxx
 	url = https://github.com/XboxDev/nxdk-libcxx.git
 	branch = nxdk
+[submodule "lib/zlib"]
+	path = lib/zlib/zlib
+	url = https://github.com/madler/zlib
```

Comment:
nit: This is the only submodule to use the short URL without `.git`

Reply:
Changed.